include/zephyr/bluetooth/bluetooth.h

```diff
@@ -2250,6 +2250,15 @@ static inline int bt_le_whitelist_clear(void)
 /**
  * @brief Set (LE) channel map.
  *
+ * Channel map format is chan_map[5] = {0xXX, 0xXX, 0xXX, 0xXX, 0xXX};
+ * chan_map[0] = 39-32
```

Comment:
As per a discussion in the Bluetooth WG meeting, we came to the conclusion that this should be:

```
* chan_map[4] = 39-32
 * chan_map[3] = 31-24
 * chan_map[2] = 23-16
 * chan_map[1] = 15-8
 * chan_map[0] = 7-0
 ```

Reply:
Or perhaps more ideally
```
 * chan_map[0] = 7-0
 * chan_map[1] = 15-8
 * chan_map[2] = 23-16
 * chan_map[3] = 31-24
 * chan_map[4] = 39-32
```